elide-core/src/main/java/com/yahoo/elide/core/PersistentResource.java

```diff
@@ -282,7 +282,7 @@ public boolean matchesId(String checkId) {
         Iterable<T> list;
         FilterScope filterScope = new FilterScope(requestScope, loadClass);
         list = tx.loadObjects(loadClass, filterScope);
-        Set<PersistentResource<T>> resources = new PersistentResourceSet(list, requestScope);
+        Set<PersistentResource<T>> resources = new PersistentResourceSet(null, list, requestScope);
```

Comment:
Can we make another constructor that doesn't take a parent instead of passing null.